### PR TITLE
Features/update UI

### PR DESCRIPTION
**Pull request by Tunent**
The previous PR should be regarded as obsolete. This PR should fix the prevous issue

### CODE REVIEW

File: frontend/src/app/main/main.component.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Inject } from '@angular/core';
```
Comment (mligtenberg) [inline frontend/src/app/main/main.component.ts:1]: I don't believe this import is used

File: frontend/src/app/main/main.component.ts [hunk 2]
```diff
@@ -15,7 +15,8 @@ export class MainComponent implements OnInit {
   subs = new Subscription();
 
   constructor(
-    private store: Store<State>
+    private store: Store<State>,
+
```
Comment (mligtenberg) [inline frontend/src/app/main/main.component.ts:19]: Could you remove this whiteline?

File: frontend/src/styles.scss [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+
+// Include the common styles for Angular Material. We include this here so that you only
+// have to load a single css file for Angular Material in your app.
+// Be sure that you only ever include this mixin once!
+@include mat.core();
+
+// Define the palettes for your theme using the Material Design palettes available in palette.scss
+// (imported above). For each palette, you can optionally specify a default, lighter, and darker
+// hue. Available color palettes: https://material.io/design/color/
+$explorer-primary: mat.define-palette(mat.$indigo-palette);
```
Comment (mligtenberg) [inline frontend/src/styles.scss:15]: Could you customize the color scheme to be in line with the colors scheme already used in the application?

See https://material.angular.io/guide/theming, for more info

**Comment by Tunent** [comment]
Updated the color scheme and resolved the comments

**Comment by mligtenberg** [comment]
I believe from a UI perspective you might need to resolve 2 thinks.
First off all the buttons seem a bit screemish now:
image

I would recommend changing it to something like this:
image

Also The spacing seems a bit off in your rows.
image
The top seems to have a smaller spacing as the bottom

**Pull request closed without merging**